src/lib/plan-frequency/PlanFrequency.spec.ts

```diff
@@ -0,0 +1,78 @@
+import '@testing-library/jest-dom';
+import { fireEvent, render } from '@testing-library/svelte';
+import PlanFrequency from './PlanFrequency.svelte';
+import * as MockDate from 'mockdate';
+
+it('should show a start date input with today as default value', () => {
+	const today = '09/13/2021';
+	MockDate.set(today);
+	const { getByLabelText } = render(PlanFrequency);
+
+	expect(getByLabelText('Start')).toHaveDisplayValue('2021-09-13');
+});
+
+it('should show an end date input with no default value', () => {
+	const { getByLabelText } = render(PlanFrequency);
+
+	expect(getByLabelText('End')).toHaveDisplayValue('');
+});
+
+it.each(['Daily', 'Weekly'])(`should show the frequency option '%s'`, (frequency) => {
+	const { getByLabelText } = render(PlanFrequency);
+
+	expect(getByLabelText(frequency)).toBeVisible();
+});
+
+it(`should have frequency option daily as default`, () => {
+	const { getByLabelText } = render(PlanFrequency);
+
+	expect(getByLabelText('Daily')).toBeChecked();
+});
+
+it.each(['Daily', 'Weekly'])(`should be able to check frequency option '%s'`, async (frequency) => {
+	const { getByLabelText } = render(PlanFrequency);
+
+	await fireEvent.click(getByLabelText(frequency));
```

Comment:
Did you consider the use of https://testing-library.com/docs/ecosystem-user-event/ instead of `fireEvent` or is that library not yet ready for Svelte?

Reply:
it looks like its not ready for svelte yet as we need to await fireEvent which is special in svelte